### PR TITLE
HAAR-339 contracted prison flag

### PR DESCRIPTION
**Pull request by AlTurner-MOJ**

### CODE REVIEW

File: src/main/resources/db/migration/V1_15__add_prison_contracted_flag.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE prison ADD COLUMN contracted BOOLEAN NOT NULL DEFAULT false;
+
+UPDATE prison SET contracted = true WHERE prison_id IN ('ACI', 'ASI', 'BZI', 'DNI', 'DGI', 'FWI', 'FBI', 'LGI', 'NLI', 'OWI', 'PBI', 'PFI', 'PRI', 'RHI', 'TSI');
```
Comment (ushkarev) [inline src/main/resources/db/migration/V1_15__add_prison_contracted_flag.sql:3]: `PYI`, "Parc A" is also a contracted prison

Comment (AlTurner-MOJ) [reply src/main/resources/db/migration/V1_15__add_prison_contracted_flag.sql:3]: Hi, we don't have Parc A in the Prison Register at the moment. I'm raising a separate ticket to get that one added

Comment (ushkarev) [reply src/main/resources/db/migration/V1_15__add_prison_contracted_flag.sql:3]: it does show in the [prod site](https://prison-register.hmpps.service.justice.gov.uk/prisons/id/PYI), but simply named as "Parc (HMYOI)"

Comment (AlTurner-MOJ) [reply src/main/resources/db/migration/V1_15__add_prison_contracted_flag.sql:3]: OK, Parc was on the ticket too, it has ID: PRI

Comment (ushkarev) [reply src/main/resources/db/migration/V1_15__add_prison_contracted_flag.sql:3]: actually, there are 2 Parc prisons (it was split earlier this year):

https://github.com/ministryofjustice/prison-register/blob/main/src/main/resources/db/migration/V1_1__insert_prisons.sql#L100

https://github.com/ministryofjustice/prison-register/blob/main/src/main/resources/db/migration/V1_7__add_hmp_parc.sql#L1

**Review by brightonsbox — APPROVED** [review]
LGTM

File: src/main/resources/db/migration/V1_15__add_prison_contracted_flag.sql (hunk 1 above)
Comment (AlTurner-MOJ) [reply src/main/resources/db/migration/V1_15__add_prison_contracted_flag.sql:3]: Ah, sorry OK - will update now - was thrown off by the name Parc A

**Review by brightonsbox — APPROVED** [review]
LGTM

**Pull request merged**